### PR TITLE
String to translate for 1.7.3

### PR DESCRIPTION
**Pull request by toutantic**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.7.3.x
| Description?  | new Strings to translate for 1.7.3
| Type?         | improvement
| Category?     | CO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | .
| How to test?  | .

#### Important guidelines

* Make sure [your local branch is up to date](https://help.github.com/articles/syncing-a-fork/) before commiting your changes!
* Your code MUST respect [our Coding Standards](http://doc.prestashop.com/display/PS16/Coding+Standards) (for code written in PHP, JavaScript, HTML/CSS/Smarty/Twig, SQL)!
* Your commit name MUST respect our [naming convention](http://doc.prestashop.com/display/PS16/How+to+write+a+commit+message)!

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/8713)

### CODE REVIEW

File: app/Resources/translations/default/ModulesGAnalyticsAdmin.xlf [hunk 1]
```diff
@@ -66,9 +66,9 @@ File: modules/ps_googleanalytics/views/templates/admin/configuration.tpl:47</not
         <note>Context:
 File: modules/ps_googleanalytics/views/templates/admin/configuration.tpl:48</note>
       </trans-unit>
-      <trans-unit id="ade6a18bb6c147db87bc9463240e455a">
-        <source>"Add to cart" actions</source>
-        <target>"Add to cart" actions</target>
+      <trans-unit id="c2083746978ad21222cccce025233610">
+        <source>Add to cart" actions</source>
+        <target>Add to cart" actions</target>
```
Comment (Quetzacoalt91) [inline app/Resources/translations/default/ModulesGAnalyticsAdmin.xlf]: Thos translation seems to be wrong

https://github.com/PrestaShop/ps_googleanalytics/blob/4258d1c42d3aadade55cfa830b77c3291b045ec3/views/templates/admin/configuration.tpl#L49

File: app/Resources/translations/default/AdminShopparametersHelp.xlf [hunk 2]
```diff
@@ -842,15 +842,15 @@ File: admin-dev/themes/default/template/controllers/referrers/helpers/form/form.
...
+        <target>_" will replace one character. If you want to use the real "_", you should type this: "\\_".</target>
         <note>Context:
 File: admin-dev/themes/default/template/controllers/referrers/helpers/form/form.tpl:63</note>
       </trans-unit>
-      <trans-unit id="900b3ed6d62723c673b427a3f6e66669">
-        <source>"%" will replace any number of characters. If you want to use the real "%", you should type this: "\\%".</source>
-        <target>"%" will replace any number of characters. If you want to use the real "%", you should type this: "\\%".</target>
+      <trans-unit id="0d3b55664eac9a05aa79b197f396f4dc">
+        <source>%" will replace any number of characters. If you want to use the real "%", you should type this: "\\%".</source>
+        <target>%" will replace any number of characters. If you want to use the real "%", you should type this: "\\%".</target>
```
Comment (Quetzacoalt91) [inline app/Resources/translations/default/AdminShopparametersHelp.xlf]: It seems all strings starting with double-quote are truncated

Comment (toutantic) [reply app/Resources/translations/default/AdminShopparametersHelp.xlf]: I believe this problem was fixed with an update to TranslationToolsBundle. I am going to check if TranslationTools (the project used to generate the translation strings) has the right dependencies

**Comment by toutantic** [comment]
I still have to fix some strings coming from modules, please wait to merge

**Comment by toutantic** [comment]
This time we have all translation strings !

**Review by Quetzacoalt91 — APPROVED** [review]
Better!

**Pull request merged**